src/main/java/edu/cornell/kfs/concur/rest/authentication/ConcurAuthenticationFilter.java

```diff
@@ -34,13 +34,18 @@ public void doFilter(ServletRequest servletRequest, ServletResponse response, Fi
 
         if (StringUtils.isBlank(authorizationHeader)) {          
             httpServletResponse.setStatus(HttpServletResponse.SC_UNAUTHORIZED);
+            LOG.debug("doFilter(): Concur authentication failed");
+            return;
         } else {
             final String encodedUserNameAndPasswordAsToken = authorizationHeader.replaceFirst(ConcurConstants.BASIC_AUTHENTICATION_SCHEME + " ", "");
 
             if (!getConcurAuthenticationService().isConcurTokenValid(encodedUserNameAndPasswordAsToken)) {
                 httpServletResponse.setStatus(HttpServletResponse.SC_UNAUTHORIZED);
+                LOG.debug("doFilter(): Concur authentication failed");
```

Comment:
Do we care to know how the login failed (no auth token versus wrong auth token)?  If so, maybe we want different debug statements.  I would be inclined to think they might be errors that we would want in the log.